codegen/src/lintdoc.rs

```diff
@@ -1608,16 +1647,24 @@ fn write_action<L: ServiceLanguage>(
 /// Parse and analyze the provided code block, and asserts that it emits
 /// exactly zero or one diagnostic depending on the value of `expect_diagnostic`.
 /// That diagnostic is then emitted as text into the `content` buffer
+#[allow(clippy::too_many_arguments)]
```

Comment:
hopefully this is acceptable, i saw it used a few other places in this file already 😅

Reply:
Yeah it's fine, we don't have a strong linting policy here :)